.github/workflows/data_library_open_data.yml

```diff
@@ -22,13 +22,57 @@ jobs:
     strategy:
```

Comment:
This is all opendata datasets. Does this seem worthwhile? Could wait. But I figure we might as well be pulling socrata data more regularly.

Main counterpoint might be to just wait until we roll out ingest - then we'll have a better record of when data is being updated

Reply:
Other argument to wait - we maybe add functionality to check for updates, and only update those that have actually changed.